src/main/java/io/gatling/demostore/Common.java

```diff
@@ -1,19 +1,21 @@
 package io.gatling.demostore;
```

Comment:
First line of first changed file: only to have a closable thread.

I think we should change the session management instead of a workaround like that.
The cart being in the session is a normal usage of the session.
The session being stored in the server is not.
Either an accessible cache (ie memcache or redis) or using session from cookie (like Play does)

First quick search on the topic:
* https://www.innoq.com/en/blog/cookie-based-spring-security-session/
* https://docs.spring.io/spring-session/reference/index.html

Reply:
https://github.com/alecalanis/session-to-cookie